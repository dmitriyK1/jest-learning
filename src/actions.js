import { createAction } from 'redux-actions';

const SAY_HI = 'SAY_HI';

const sayHi = createAction(SAY_HI);

export { sayHi };
