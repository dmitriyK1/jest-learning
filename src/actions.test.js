import { sayHi } from "./actions";

it('sayHi matches snapshot', () => {
  expect(sayHi()).toMatchSnapshot();
});
