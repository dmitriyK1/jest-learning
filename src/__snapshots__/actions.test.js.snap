// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sayHi matches snapshot 1`] = `
Object {
  "type": "SAY_HI",
}
`;
